src/components/Web3Connect/index.tsx

```diff
@@ -22,16 +37,47 @@ export default function Web3Connect({ color = 'gray', size = 'sm', className = '
       {error instanceof UnsupportedChainIdError ? i18n._(t`You are on the wrong network`) : i18n._(t`Error`)}
     </div>
   ) : (
-    <Button
-      id="connect-wallet"
-      onClick={toggleWalletModal}
-      variant="outlined"
-      color={color}
-      className={className}
-      size={size}
-      {...rest}
-    >
-      {i18n._(t`Connect to a wallet`)}
-    </Button>
+    <>
+      <Button
+        id="connect-wallet"
+        onClick={handleConnectBtnClick}
+        variant="outlined"
+        color={color}
+        className={className}
+        size={size}
+        {...rest}
+      >
+        {i18n._(t`Connect to a wallet`)}
+      </Button>
+      <InstructionModal
+        isOpen={isInstructionModalOpen}
+        onDismiss={onInstructionModalDismiss}
+        instruction={
+          <div>
+            {deviceInfo.isChrome && deviceInfo.isDesktop ? (
+              <>
+                <p>
+                  {i18n._(t`Chrome is supported, but first you need to install`)}
+                  <a
+                    className="text-blue"
+                    href="https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn"
+                  >
+                    {' '}
+                    chrome metatask extension{' '}
+                  </a>
+                </p>
+              </>
+            ) : (
+              <>
+                {i18n._(t`${deviceInfo.browserName} is not supported`)} {t`please install`}{' '}
+                <a className="text-blue" href="https://metamask.io/">
+                  MetaTask App
```

Comment:
metatask -> metamask

Reply:
also these should be in translation (  i18n._t(``)  )

Reply:
fixed.